lib/core/network/protocols/httpwsProtocol.js

```diff
@@ -502,7 +502,7 @@ class HttpWsProtocol extends Protocol {
 
     if (
       contentType &&
-      !HTTP_ALLOWED_CONTENT_TYPES.some((allowed) =>
+      !this.httpConfig.opts.allowedContentTypes.some((allowed) =>
```

Comment:
Why this variable has changed ?

Reply:
The default allowed content types and the additional ones from the config are merged into this variable L1197